templates/integration/__init__.py

```diff
@@ -7,7 +7,7 @@
 ~~~~~~~~
 
 The foo integration is enabled automatically when using
-:ref:`ddtrace-run <ddtracerun>` or :func:`patch_all() <ddtrace.patch_all>`.
+:ref:`ddtrace-run <ddtracerun>` or :mod:`import ddtrace.auto <ddtrace.auto>`.
```

Comment:
a few places use `:ref:` and a only a few use `:mod:`. I didn't notice a pattern for why one is used vs the other, is it intentional?

Reply:
thanks for catching that. those oversights are the result of malformed `sed` commands.